app/src/main/java/dev/arkbuilders/arkmemo/preferences/MemoPreferences.kt

```diff
@@ -8,4 +8,8 @@ interface MemoPreferences {
     fun getPath(): String
 
     fun getNotesStorage(): Path
+
+    fun storeCrashReportEnabled(bool: Boolean)
```

Comment:
nit: Could we have a more meaningful name for the param? `bool` is not the best one, I guess.